entities/auctioneer.py

```diff
@@ -49,3 +54,6 @@ def _registration(self, type):
                 return key
 
         return max_key + 1  # new key, one greater than the last
+
+    def list_shipment(self, bid):
+        self.auctionable_shipments.append(bid)
```

Comment:
This does not need a separate function, just use the register member function (you don't need a new data structure for this).